python/cudf/cudf/core/dataframe.py

```diff
@@ -3748,20 +3772,41 @@ def any(self, bool_only=None, **kwargs):
             )
         return self._apply_support_method("any", **kwargs)
 
-    def _apply_support_method(self, method, *args, **kwargs):
-        result = [
-            getattr(self[col], method)(*args, **kwargs)
-            for col in self._cols.keys()
-        ]
-        if isinstance(result[0], Series):
-            support_result = result
-            result = DataFrame()
-            for idx, col in enumerate(self._cols.keys()):
-                result[col] = support_result[idx]
-        else:
-            result = Series(result)
-            result = result.set_index(self._cols.keys())
-        return result
+    def _apply_support_method(self, method, axis=0, *args, **kwargs):
+        assert axis in (None, 0, 1)
+
+        if axis in (None, 0):
+            result = [
+                getattr(self[col], method)(*args, **kwargs)
+                for col in self._cols.keys()
+            ]
+
+            if isinstance(result[0], Series):
+                support_result = result
+                result = DataFrame()
+                for idx, col in enumerate(self._cols.keys()):
+                    result[col] = support_result[idx]
+            else:
+                result = Series(result)
+                result = result.set_index(self._cols.keys())
+            return result
+
+        elif axis == 1:
+            # for dask metadata compatibility
+            kwargs.pop("skipna", None)
+
+            prepared = self._prepare_for_rowwise_op()
+            arr = cp.asarray(prepared.as_gpu_matrix())
```

Comment:
@brandon-b-miller are we okay with just using CuPy wherever now or are we still checking for whether it's installed or not?

Reply:
@kkraus14 now that the updates to the conda `yml` files are merged it should be ok to just import and call CuPy anywhere in the code. In fact the code that does the checking should probably be removed now that we have a way to stitch the proper cupy/numba versions together.

Reply:
Love it!